packages/react/src/analytics/integrations/Guardian/providers/Riskified/Riskified.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  EventData,
+  integrations,
+  LoadIntegrationEventData,
+  StrippedDownAnalytics,
+  TrackTypes,
+  TrackTypesValues,
+} from '@farfetch/blackout-analytics';
+import get from 'lodash/get';
+import type { RiskifiedIntegrationOptions } from './types';
+
+class Riskified extends integrations.Integration<RiskifiedIntegrationOptions> {
+  isReady: boolean;
+  siteId: string;
+  baseUrl: string;
+
+  /**
+   * Creates an instance of Riskified and sets the script on the page.
+   *
+   * @param options - Custom options for the integration.
+   * @param loadData - Analytics's load event data.
+   * @param analytics - Stripped down analytics instance.
+   */
+  constructor(
+    options: RiskifiedIntegrationOptions,
+    loadData: LoadIntegrationEventData,
+    analytics: StrippedDownAnalytics,
+  ) {
+    super(options, loadData, analytics);
+
+    this.isReady = false;
+    this.siteId = options.siteId || 'farfetch.com';
+    this.baseUrl = `https://beacon.riskified.com?shop=${this.siteId}&sid=`;
+
+    this.setupScript(loadData);
+  }
+
+  /**
+   * Returns true due to being a required integration - No need to check for consent.
+   *
+   * @returns A value indicating if the integration should load.
+   */
+  static override shouldLoad(): boolean {
+    return true;
+  }
+
+  /**
+   * Creates the script based on the correlation `id` and appends it on the page.
+   * Once the script is on the page, it will automatically call `RISKX.go()`.
```

Comment:
I think this description is not correct. Once the script is on the page, it will not call `RISKX.go` but will only set the `isReady` property to true. Can you check, please?

Reply:
fixed